static/schemas/obms/snmp-obm-service.json

```diff
@@ -1,9 +1,10 @@
 {
-    "title": "snmp-ibm-service",
+    "title": "snmp-obm-service",
     "definitions": {
-        "Ibm": {
-            "description": "IBM settings",
+        "ObmPatch": {
```

Comment:
@BillyAbildgaard please help to check whether "ibm" is suitable to change to "obm"